TeamCode/src/main/java/org/firstinspires/ftc/teamcode/roadrunner/drive/DriveConstants.java

```diff
@@ -43,19 +43,19 @@ public class DriveConstants {
      * angular distances although most angular parameters are wrapped in Math.toRadians() for
      * convenience. Make sure to exclude any gear ratio included in MOTOR_CONFIG from GEAR_RATIO.
      */
-    public static double WHEEL_RADIUS = 2; // in
-    public static double GEAR_RATIO = 1; // output (wheel) speed / input (motor) speed
-    public static double TRACK_WIDTH = 1; // in
+    public static double WHEEL_RADIUS = 1.89; // in, 48mm
+    public static double GEAR_RATIO = 19.2; // output (wheel) speed / input (motor) speed
```

Comment:
Some of the constants in DriveConstants are tuned for motor encoders because I did that first, but the actually used value should be in [TwoWheelTrackingLocalizer.java](https://github.com/Stonks3141/tungsteel-powerplay/blob/main/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/roadrunner/drive/TwoWheelTrackingLocalizer.java#L38).